docs/simplification.md

```diff
@@ -12,89 +18,83 @@ The AsyncAPI Model SDK supports simplification of:
 - [Required](#determining-the-required-properties-for-the-model)
 - [Extend](#determining-the-extend-for-the-model)
 
+<br/>
+
 ## Determining the type for the model
 
-In order to determine all the possible types a schema can be, we both infer and use existing definitions of types, however we need to define a precedence for JSON Schema keywords for in which order they types are inferred or determined.
+In order to determine all the possible types a model can be based on schema, we both infer and use existing definitions of types, however we need to define a precedence for JSON Schema keywords for in which order they types are inferred or determined.
 
-### Precedence
+**Precedence of JSON Schema keywords**
 
 The precedence of keywords are in which order we infer or determine types in. Notice it goes form left to right meaning first we set the type using the `type` keyword and then it moves right applying them one by one. The following are the precedence for determining types:
```

Comment:
Most of this would be already mentioned in https://github.com/asyncapi/generator-model-sdk/pull/87/files#r580499801 so I think it can partially be removed.